docs/dev-tools/zombienet/overview.md

```diff
@@ -319,286 +519,975 @@ For example, the following configuration file defines a minimal example for the
-    | `substrate_cli_args_version` | 0 \| 1           | By default zombienet evaluates the binary and sets the correct version. Set this key directly to skip overhead. | -                    |
-    | `command_with_args`          | String           | Overrides both command and arguments for the collator                                                           | -                    |
-    | `env`                        | Array of objects | Environment variables to set in the container for the collator                                                  | -                    |
-    | `env.name`                   | String           | Name of the environment variable                                                                                | -                    |
-    | `env.value`                  | String \| Number | Value of the environment variable                                                                               | -                    |
-    | `keystore_key_types`         | String           | Defines which keystore keys should be created. For more details, refer to additional documentation              | -                    |
-
-    For example, the following configuration file defines a minimal example for the collator:
-
-    === "collator-example.toml"
-        ```toml
-        [parachain]
-        id = 100
-        add_to_genesis = true
-        cumulus_based = true
-        genesis_wasm_path = "/path/to/wasm"
-        genesis_state_path = "/path/to/state"
+    === "Type" 
 
-        [[parachain.collators]]
-        name = "alice"
-        image = "polkadot-parachain"
-        command = "polkadot-parachain"
-        ...
-        ```
+        - String
+        - Should be the plain version to allow customizations
 
-    === "collator-example.json"
-        ```json
-        {
-          "parachain": {
-            "id": 100,
-            "add_to_genesis": true,
-            "cumulus_based": true,
-            "genesis_wasm_path": "/path/to/wasm",
-            "genesis_state_path": "/path/to/state",
-            "collators": [
-              {
-                "name": "alice",
-                "image": "polkadot-parachain",
-                "command": "polkadot-parachain",
+    === "Default Value"
+
+        - none
+
+??? function "`chain_spec_command` - command to generate the chain spec. It can't be used in combination with `chain_spec_path`"
+
+    === "Type" 
+
+        - String
+
+    === "Default Value"
+
+        - none
+
+??? function "`default_args` - an array of arguments to use as default to pass to the command"
+
+    === "Type" 
+
+        - Array of strings
+
+    === "Default Value"
+
+        - none
+
+??? function "`default_substrate_cli_args_version` - set the Substrate CLI args version"
+
+    === "Type" 
+
+        - 0 \| 1 \| 2
```

Comment:
this isn't a type, they are values. Looks like the type is actually `SubstrateCliArgsVersion` https://github.com/paritytech/zombienet/blob/71d24990fa2e046fa576686a163dd66c80d42002/javascript/packages/orchestrator/src/configTypes.ts#L35

and this is an enum https://github.com/paritytech/zombienet/blob/71d24990fa2e046fa576686a163dd66c80d42002/javascript/packages/orchestrator/src/sharedTypes.ts#L3